README.md

```diff
@@ -29,7 +29,7 @@ Install
 -------
 
 ```bash
-$ composer install
+$ composer create-project sculpin/blog-skeleton my-blog
```

Comment:
this requires the package to be registered with packagist. can one of the maintainers please do that?

Reply:
#47 is fixed